data/INDY/projects.yaml

```diff
@@ -14,16 +14,24 @@ projects:
       - type: Paper
         title: Fast and Accurate Inference of Plackett–Luce Models
         url: https://infoscience.epfl.ch/record/213486/files/fastinference.pdf
+        notes:
+          - label: Published in
+            text: proceedings of NeurIPS 28, 2015
+            url: https://proceedings.neurips.cc/paper/2015
       - type: Paper
         title: "ChoiceRank: Identifying Preferences from Node Traffic in Networks"
         url: https://infoscience.epfl.ch/record/229164/files/choicerank.pdf
+        notes:
+          - label: Published in
+            text: proceedings of Machine Learning Research Volume 70, August 2017
+            url: https://proceedings.mlr.press/v70/
```

Comment:
SSL error? Cert is only valid for github and co

Reply:
yes, plain old http looks better there...